README.md

```diff
@@ -96,7 +96,18 @@ To quickly start using Passkeys in your Django project, follow these steps:
 
    ```
 
-5. Add the registration code to your logged-in user template.
+5. Add `django_otp_webauthn.backends.WebAuthnBackend` to your `AUTHENTICATION_BACKENDS` in your Django settings. This is only required if you are using passkeys for
+passwordless authentication. If another authentication method is required first then this is not necessary.
+
+    ```python
+    AUTHENTICATION_BACKENDS = [
+        ...
+        "django_otp_webauthn.backends.WebAuthnBackend",
+        ...
+    ]
+    ```
```

Comment:
**Nitpick:** slight rewording which I believe to be clearer

```suggestion
5. Add `django_otp_webauthn.backends.WebAuthnBackend` to `AUTHENTICATION_BACKENDS` in your Django settings. This step is required to make 'passwordless authentication' work.

If you are exclusively using Passkeys as a secondary verification step, you don't have to add this backend.

    ```python
    AUTHENTICATION_BACKENDS = [
        ...
        "django_otp_webauthn.backends.WebAuthnBackend",
        ...
    ]
    ```
```